### PR TITLE
Reduce inter-template redundancy

### PR DESCRIPTION
**Pull request by mbuechse**

### CODE REVIEW

**Comment by mbuechse** [comment]
Screenshot from 2024-09-06 10-10-25

File: compliance-monitor/templates/details.md.j2 [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
 | testcase id | result | description |
 |---|---|---|
 {% for testcase_id in target_result.testcases -%}
 {% set testcase = version_result.testcases[testcase_id] -%}
-{% set result = version_result.results[testcase_id].result if testcase_id in version_result.results else 0 -%}
-| {{ testcase.id }} | {{ result | verdict }} | {{ testcase.description | trim }} |
-{% endfor -%}
-|---|---|---|
-
+{% set res = version_result.results[testcase_id] if testcase_id in version_result.results else dict(result=0) -%}
```
Comment (artificial-intelligence) [inline compliance-monitor/templates/details.md.j2:20]: I'm not very familiar with this table, but shouldn't this be:
```suggestion
{% set res = version_result.results[testcase_id].result if testcase_id in version_result.results else dict(result=0) -%}
```

**Review by artificial-intelligence — CHANGES_REQUESTED** [review]
LGTM appart from one error, I believe. Can't comment on the HTML stuff, my frontend knowledge is roughly 20 years old.

File: compliance-monitor/templates/details.md.j2 (hunk 1 above)
Comment (mbuechse) [reply compliance-monitor/templates/details.md.j2:20]: No, it's correct the way it's now. See the else branch

Comment (mbuechse) [reply compliance-monitor/templates/details.md.j2:20]: (I did test it!)

**Review by artificial-intelligence — APPROVED** [review]

**Pull request merged**